src/content/content.js

```
@@ -1,146 +1,477 @@
+          elementSelector = rule ? rule.codeSelector : 'pre';
+        }
+        
+        logDebug(`Using selector from rulesList: ${elementSelector}`);
+        
+        // Convert selector string to actual DOM operation
+        let element = null;
+        if (elementSelector) {
+          if (elementSelector.includes('document.querySelector')) {
+            // Extract the selector string from the code and execute it
+            const selectorMatch = elementSelector.match(/document\.querySelector\(['"]([^'"]+)['"]\)/);
+            if (selectorMatch && selectorMatch[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)